src/os/ui/slick/slickgrid.js

```diff
@@ -1107,10 +1107,15 @@ os.ui.slick.SlickGridCtrl.prototype.onContextMenu_ = function(event, opt_positio
         os.ui.openMenu(menu, {x: menuX, y: menuY});
       }
     } else if (menu instanceof os.ui.menu.Menu) {
+      const rect = this.element[0].getBoundingClientRect();
+
+      menuX -= rect['left'];
```

Comment:
This shouldn't require bracket notation. If you were seeing issues in compile mode that is probably indicative of another issue.

Reply:
I removed the bracket notation and it still works in compiled mode.